dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/ActivityName.ts

```diff
@@ -8,6 +9,20 @@ export function activityApiNameToHumanizedName(target: string): string {
   return assertDefined(Object.entries(ActivityName).find(([key]) => key == target))[1];
 }
 
+/**
```

Comment:
If this is now used for eu taxonomy and gdv/vöp, it should be filled under a common folder and not under euTaxonomy.

Reply:
Ok, looks good now